tests/saw/Makefile

```diff
@@ -103,23 +102,19 @@ failure-tests : bitcode
 
 # We're just making separate prefix targets for each saw script we want to do
 # negative tests on
-failure_tests/sha_%.log : bitcode/sha_%.bc
-        #this might not be necessary
```

Comment:
Do we know why these tests were removed? Were they ever added back in some form?

Reply:
I didn't find any context in the PR they were removed from. The removed test includes `HMAC_driver.saw`, which is now `verify_HMAC.saw`. However, it calls a function from there that no longer exists. If we were to bring the tests back I think some work would have to be done to update them.